### PR TITLE
update Notify to exports.qbx_core.Notify

### PR DESCRIPTION
**Pull request by bicatcho11**
## Description

update Notify to exports.qbx_core.Notify

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [ ] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -8,7 +8,7 @@ local function setJailStatus(src, jailTime)
     if jailTime > 0 then
         if player.PlayerData.job.name ~= "unemployed" then
             player.Functions.SetJob("unemployed")
-            TriggerClientEvent('QBCore:Notify', src, locale("info.lost_job"))
+            exports.qbx_core.Notify(src, locale("info.lost_job"))
```
Comment (Manason) [inline server/main.lua]:
```suggestion
            exports.qbx_core:Notify(src, locale("info.lost_job"))
```

File: server/main.lua [hunk 2]
```diff
@@ -106,7 +106,7 @@ RegisterNetEvent('qbx_prison:server:completedJob', function()
     local chance = math.random(100)
     if chance > Config.Jobs.electrician.rewardChance then return end
     if not player.Functions.AddItem(Config.Jobs.electrician.reward, 1) then return end
-    TriggerClientEvent('QBCore:Notify', src, locale('success.found_item', Config.Jobs.electrician.reward), 'success')
+    exports.qbx_core.Notify(src, locale('success.found_item', Config.Jobs.electrician.reward), 'success')
```
Comment (Manason) [inline server/main.lua]:
```suggestion
    exports.qbx_core:Notify(src, locale('success.found_item', Config.Jobs.electrician.reward), 'success')
```

**Review by Manason — APPROVED** [review]

**Review by Manason — APPROVED** [review]

**Review by solareon — CHANGES_REQUESTED** [review]
If we are adding locale strings please add them to all the locales to prevent issues.

File: client/prisonbreak.lua [hunk 3]
```diff
@@ -151,9 +151,9 @@ local function createGateZones()
             end,
             inside = function()
                 if securityLockdown then
-                    qbx.drawText3d({ text = "~r~SYSTEM LOCKDOWN", coords = gates[i].coords })
+                    qbx.drawText3d({ text = "~r~" .. locale('info.system_lockdown'), coords = gates[i].coords })
```
Comment (BerkieBb) [inline client/prisonbreak.lua:154]: This ~r~ can be added to the locales as well

**Review by BerkieBb — APPROVED** [review]

File: locales/ar.json [hunk 4]
```diff
@@ -1,24 +1,39 @@
...
+    "seized_property": "تم حجز ممتلكاتك، ستستعيدها عند انتهاء وقتك..",
+    "target_canteen_option": "احصل على الطعام",
+    "target_freedom_option": "تحقق من الوقت",
+    "timeleft": "لديك لا تزال... %s شهرًا",
+    "work_blip": "العمل في السجن",
+    "working_electricity": "توصيل الأسلاك"
   },
   "success": {
-    "found_item": "لقد عثرت على هاتف",
+    "found_item": "لقد وجدت %s..",
```
Comment (solareon) [inline locales/ar.json:34]: nice catch

**Review by solareon — APPROVED** [review]

**Pull request merged**